spec/features/user_sees_list_of_books_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+feature 'User sees a list of books' do
+  scenario 'successfully' do
+    books = create_pair(:book)
+
+    visit books_path
```

Comment:
I would delete the lines between the create and visit parts of this. https://robots.thoughtbot.com/four-phase-test